src/test/java/org/folio/marc4ld/mapper/Bibframe2Marc010IT.java

```diff
@@ -69,6 +69,22 @@ void should_map_all_subfields_z() {
       .isEqualTo(expectedMarc);
   }
 
+  @Test
+  void should_ignore_lccn_with_invalid_status() {
+    // given
+    var expectedMarc = loadResourceAsString("fields/marc_empty.jsonl");
+    var resource = createInstanceWithLccn(
+      lccn("11111111", "invalid_status")
+    );
+
+    // when
+    var result = bibframe2MarcMapper.toMarcJson(resource);
+
+    // then
+    assertThat(result)
+      .isEqualTo(expectedMarc);
+  }
```

Comment:
Empty Marc is too simple an object to keep separately.
suggest to replace to a simple assertion, like this
```
    assertThat(result.getfields)
      .isEmpty
```
It is easier to support

Reply:
Done